modules/overview/pages/tokens.adoc

```diff
@@ -0,0 +1,154 @@
+include::{mod-loc}attributes.adoc[]
+
+:context: skupper-tokens
+//Category: skupper-tokens
+// Type: assembly
+[id='using-skupper-tokens-{context}']
+
+= Using Skupper tokens
+
+[role="system:abstract"]
+Skupper tokens allow you create links between sites.
+You create a token on one site and use that token from the other site to create a link between the two sites.
+
+NOTE: Although the linking process is directional, a Skupper link allows communication in both directions.
+
+If both sites are equally accessible, for example two public clouds, then it not important where the token is created.
+However, when using a token, the site you are linking to must be accessible from the site you are linking from.
+For example, if you are creating a {service-network} using both a public and private cluster, you must create the token on the public cluster and use the token from the private cluster.
+
+There are two types of Skupper token:
+
+Claim token (default):: 
++
+--
+A claim token can be restricted by:
+
+* time - preventing reuse after a specified period.
+* usage - preventing multiple links being created from a single token.
+--
+
+Cert token ::
++
+--
+A cert token can be reused to create many links from many sites.
+--
+
+// Type: procedure
+[id='creating-claim-tokens-{context}']
+== Creating claim tokens
+
+[role="system:abstract"]
+Creating claim tokens allows you to restrict access to the {service-network}
+
+.Procedure
+
+. Log into the cluster.
+
+. Change to the namespace associated with the site.
+
+. Create a claim token, for example:
++
+--
+[source,bash,options="nowrap"]
+----
+$ skupper token create $HOME/secret.yaml --expiry 30m0s --uses 2 -t claim
+----
+
+NOTE: Claim tokens are the default, the `-t claim` section of the command is unnecessary.
+
+--expiry:: The amount of time the token is valid in minutes and seconds, default `15m0s`.
+--uses:: The number of times you can use the token to create a  link, default `1`.
+--
+
+.Additional information
+
+* See the {CliLink} for information on using the token to create links.
+
+// Type: procedure
+[id='creating-claim-tokens-{context}']
+== Creating cert tokens
+
+[role="system:abstract"]
+Creating cert tokens allows you create many links to the {service-network} from different sites without restrictions.
+
+.Procedure
+
+. Log into the cluster.
+
+. Change to the namespace associated with the site.
+
+. Create a cert token:
++
+--
+[source,bash,options="nowrap"]
+----
+$ skupper token create $HOME/secret.yaml -t cert
+----
+
+NOTE: Cert tokens are always valid, and can be reused indefinitely.
+--
```

Comment:
There is now the ability to revoke-access on a site which invalidates any previously issued token of any kind.

Reply:
added link